include/uv-win.h

```diff
@@ -366,8 +366,8 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
     struct {                                                                  \
       OVERLAPPED overlapped;                                                  \
       size_t queued_bytes;                                                    \
-    };                                                                        \
-  };                                                                          \
+    } io;                                                                     \
+  } req;                                                                      \
```

Comment:
@saghul Actually 'data' is already used by the `UV_REQ_FIELDS` as a void pointer, so it'd be a duplicate. I agree another name might be better though.